main.tf

```diff
@@ -299,11 +299,31 @@ resource "aws_appautoscaling_policy" "autoscaling" {
 
   target_tracking_scaling_policy_configuration {
     predefined_metric_specification {
-      predefined_metric_type = "RDSReaderAverageCPUUtilization"
+      predefined_metric_type = "${var.predefined_metric_type_cpu}"
     }
 
     scale_in_cooldown  = "${var.replica_scale_in_cooldown}"
     scale_out_cooldown = "${var.replica_scale_out_cooldown}"
     target_value       = "${var.replica_scale_cpu}"
   }
 }
+
+resource "aws_appautoscaling_policy" "autoscaling_connections" {
+  count              = "${var.replica_scale_enabled ? 1 : 0}"
+  depends_on         = ["aws_appautoscaling_target.autoscaling"]
+  name               = "target-metric"
+  policy_type        = "TargetTrackingScaling"
+  resource_id        = "cluster:${aws_rds_cluster.default.cluster_identifier}"
+  scalable_dimension = "rds:cluster:ReadReplicaCount"
+  service_namespace  = "rds"
+
+  target_tracking_scaling_policy_configuration {
+    predefined_metric_specification {
+      predefined_metric_type = "${var.predefined_metric_type_connections}"
```

Comment:
RDSReaderAverageCPUUtilization and RDSReaderAverageDatabaseConnections are the only metrics types you can use. I think it does not need a variable

Reply:
Also, I have removed the `predefined_metric_type_connections` here:
https://github.com/claranet/terraform-aws-aurora/pull/38/commits/e424f3e25f6e799d0154ae13e73bd71a7bee68bd#diff-7a370d8342e7203b805911c92454f0f4R322